lua/damagelogs/server/discord.lua

```diff
@@ -73,7 +73,7 @@ function Damagelog:DiscordMessage(discordUpdate)
     }
 
 
-    if discordUpdate.responseMessage != nil then
+    if discordUpdate.responseMessage ~= nil then
```

Comment:
Just in-case you weren't aware, Garry's Mod Lua supports some non-standard operators like `!`.
These will work fine in-game, and using a GLua linter in your code editor, but might not work using a stand Lua linter in your code editor.
https://wiki.facepunch.com/gmod/Specific_Operators

I'm happy to replace the non-standard operators, but I have no objection to using them.

Reply:
Ya Malivil, one of the other mod makers I work with at times let me know that when I told him about tbis PR. It's still better to use proper Lua conventions even if Gmod is weird though